pycc/ccwfn.py

```diff
@@ -10,11 +10,12 @@
 import time
 import numpy as np
 import torch
-from .utils import helper_diis, cc_contract
-from .hamiltonian import Hamiltonian
-from .local import Local
-from .cctriples import t_tjl, t3c_ijk
-from .lccwfn import lccwfn
+from utils import helper_diis, cc_contract #helper_ldiis
+from hamiltonian import Hamiltonian
+from local import Local
+from cctriples import t_tjl, t3c_ijk
+from lccwfn import lccwfn
+from debug import Debug 
```

Comment:
Can you help me to understand what's happening here?  Are you temporarily changing the interface to run the code directly?